logic/src/androidMain/kotlin/com/wire/kalium/logic/data/asset/KaliumFileSystemImpl.kt

```diff
@@ -115,4 +128,11 @@ actual class KaliumFileSystemImpl actual constructor(
      * Fetches the persistent [Path] of the current user's avatar in the [KaliumFileSystem]
      */
     override fun selfUserAvatarPath(): Path = providePersistentAssetPath("self_user_avatar.jpg")
+
+    /**
+     * Provides a list of paths found in the given [dir] path from where the call is being invoked.
+     * @param dir the path from where the list of paths will be fetched
+     * @return the list of paths found.
+     */
+    override suspend fun list(dir: Path): List<Path> = SYSTEM.list(dir)
```

Comment:
maybe a more meaningfull name ?

Reply:
more meaningful name for the method?

Reply:
changed it for `listDirectories`

Reply:
I think directories would be sufficient honestly